README.md

```diff
@@ -1,27 +1,78 @@
-# goitneo-python-hw-3-group-3
+# CLI Personal Assistant
 
 ## Description
 A CLI organizer that can help store contacts' phone numbers and birthday dates.
 
-
 ## Get Started
 
-Run `pip install personal_assistant` to install the package and then `get-started` to get started. The list of available commands is as follows:
+Run `pip install personal_assistant` to install the package and then `get-started` to get started.<br><br>
+
+**The list of available commands is as follows:**
 
 - `hi` or `hello` or `greeting` - to say hi to your favorite PA!
+- `contacts` - to enter the address book interface;
+- `notes` - to enter the notes interface;
+- `close` or `exit` - to exit the personal assistant;
+
+## Interface Address book
+
 - `add name phone` - to add a acontact and its phone number;
```

Comment:
a typo in acontact

Reply:
@Nataliia-Aldushyna Install the "Code Spell Checker" extension for VS Code. It will help you fix mistakes in words

Reply:
thank you :) @DmytroVasylenko6